sale_promotion_domain_product_discount/tests/test_domain_product_discount.py

```diff
@@ -0,0 +1,179 @@
+from odoo.tests import tagged
+from odoo.tests.common import SavepointCase
+
+
+@tagged("post_install", "-at_install")
+class CetmixTestSaleCouponProgram(SavepointCase):
+    def setUp(self):
+        super(CetmixTestSaleCouponProgram, self).setUp()
+        self.largeCabinet = self.env["product.product"].create(
```

Comment:
`SavepointCase` uses the `cls` object, so you should change your case declaration like:

```suggestion
        cls.largeCabinet = cls.env["product.product"].create(
```